tests/framework/environment/environment.go

```diff
@@ -50,4 +57,12 @@ func init() {
 	flag.StringVar(&Env.LoadSize, "load_size", "medium", "load size for each thread performing operations - small,medium or large.")
 	flag.BoolVar(&Env.EnableChaos, "enable_chaos", false, "used to determine if random pods in a namespace are to be killed during load test.")
 	flag.StringVar(&Env.Logs, "logs", "", "Gather rook logs, eg - all")
+	flag.BoolVar(&Env.RookTestOpenshift, "rook_test_openshift", false, "used to determine if running agains openshift ENV.")
+	flag.IntVar(&Env.RetryLoopNumber, "retry_loop", 40, "number of attempts for retry in the loop.")
+	flag.StringVar(&Env.RookBaseTestDir, "rook_base_test_dir", "", "base test dir of rook.")
+	flag.BoolVar(&Env.SkipCephHelmSuite, "skip_ceph_helm_suite", false, "skip CephHelmSuite")
+	flag.BoolVar(&Env.SkipCephMultiClusterDeploySuite, "skip_ceph_multi_cluster_deploy_suite", false, "skip CephMultiClusterDeploySuite")
+	flag.BoolVar(&Env.SkipCephUpgradeSuite, "skip_ceph_upgrade_suite", false, "skip CephUpgradeSuite")
+	flag.BoolVar(&Env.SkipCephBlockMountUnMountSuite, "skip_ceph_block_mount_unmount_suite", false, "skip CephBlockMountUnMountSuite")
+	flag.Parse()
```

Comment:
@travisn I was kind of surprised that this `flag.Parse()` wasn't called before cause without this call I couldn't get value of any passed arguments

Reply:
as this package name was `installer` earlier it was called here https://github.com/rook/rook/blob/ba74cc9361bbccd2d9f94411b826538967bf11d3/tests/framework/installer/ceph_installer.go#L579

Reply:
@BlaineEXE The intention is to run all tests starting with: TestCeph
Before the rebase and align with Travis comments we had passing those two test suites:
- TestCephSmokeSuite
- TestCephBlockCreateSuite

I had the conditions different way and run just those tests which were already unblocked if openshift testing parameter passed. But then @travisn commented to use this approach: https://github.com/rook/rook/pull/3523#discussion_r315422213 . So I did for each of those tests suites which do not pass the parameter to have possibility to skip them as was requested by Travis.